.asyncapi-tool.yaml

```diff
@@ -0,0 +1,13 @@
+title: AsyncAPI Studio
+description: Visually design your AsyncAPI files and event-driven architecture.
+links:
+  websiteUrl: https://studio.asyncapi.com
+filters:
+  language: JavaScript
+  technology:
+    - React
+    - TypeScript
+  categories:
```

Comment:
since `category` is a grouping element, imho better to have `editor` only for studio. It is not platform, and UI is kinda the same as editor. We can later have category of editors, and IDE editor plugins could be listed there too